cmd/vic-machine/common/dns.go

```diff
@@ -0,0 +1,56 @@
+package common
```

Comment:
@anchal-agrawal we should see if we should adjust the Makefile to pass check-headers.sh the fix argument so that the make all would add copyrights for us, though I wonder how it would work for new files, because I think it puts the range by default.